src/libcharon/plugins/vici/vici_cred.c

```diff
@@ -482,7 +483,7 @@ CALLBACK(load_shared, vici_message_t*,
 		DBG1(DBG_CFG, "loaded %N shared key for: %s",
 			 shared_key_type_names, type, buf);
 	}
-
+	DBG4(DBG_CFG, "Loading shared key of type %N for identities '%s': %#B", shared_key_type_names, type, buf, &data);
```

Comment:
Hmmmh. I dislike it because the way I did it, one can just search for the identity or shared key type name and find the line with the verbatim secret. If we just log "key: %#B", then one has to scroll up.

Reply:
I guess you'd usually have to scroll down one line (from the identity to the key), which really doesn't seem like an issue for something that only rarely will be necessary.